### PR TITLE
Create Account iban fix

### PR DESCRIPTION
**Pull request by huseynovramilx**
Modified create account endpoint to return iban,
Modified saveData to save iban,
Fixed the bank constructor to not duplicate bank account

### CODE REVIEW

File: src/main/java/files/customers.txt [hunk 1]
```diff
@@ -1,2 +1,9 @@
-e2f7bc4c-ef2a-466e-b8a5-d24f48b13afe,John,Doe,john@doe.ee,pass1234
-1eaf6f4c-4f92-4e04-bd87-c82be03265de,Bob,Jackson,bobby@tt.ee,secret
\ No newline at end of file
+21eb2d43-1a04-430f-b8a5-5612d60a883c,gaASDSSDF,DFSDDSA,ASDSD@ASDbuASmdDSDe.com,MPO4yynMyN
+a70d08d5-ec28-467a-adaf-6845d3e3ea04,Egjs,Bumde,Egjs@bumde.com,kJOIgFvIOW
+5d021ce0-a6c8-45e9-ac0f-c2d6d7785ddf,gaASDSSDF,DFSDDSA,SD@ASDbuASmdDSDe.com,8o45xEnM0B
+6e3cd55b-b55d-4c1c-a631-b801fc915401,gaASDSSDF,DFSDDSA,SD@buASmdDSDe.com,XwglsWexlk
+91c6fc4b-e762-4f6d-9c0b-4598162847fd,gaSDF,ASDGASD,SD@bumdSDe.com,nKmNEdLM2M
+8bd34921-c54a-45db-b0b0-fd24d38a9a40,gaSSDF,DFDSA,SD@bumdDSDe.com,vebptuDwoS
+67b83f8f-3613-497c-9186-503f6a3679d2,gaASDSSDF,DFSDDSA,ASDSD@SD.com,sxIY3VEQYe
```
Comment (ira-halenok) [inline src/main/java/files/customers.txt:7]: There is no need to push all these customers to `main`.

**Review by ira-halenok — CHANGES_REQUESTED** [review]

**Comment by ira-halenok** [comment]
@huseynovramilx we also need to adjust the customers/transactions endpoint and accept IBAN code from UI, load accounts by IBAN and then perform the transaction.
And also there is one more serializer: `CustomerSerializer`, it works with the FE, so we need to add adjustments there, as well.

**Comment by JonasBerx** [comment]
merged the generated diagrams already. Dont commit the generated diagrams to the branches. Only code

**Comment by ira-halenok** [comment]
@JonasBerx need your approval on whether we can merge the changes to generated diagrams

**Review by ira-halenok — APPROVED** [review]

**Pull request merged**